libs/partners/google-vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -347,6 +349,11 @@ def _generate(
         Raises:
             ValueError: if the last message in the list is not from human.
         """
+        from google.cloud.aiplatform_v1beta1.types import (
+            content as gapic_content_types,
+        )
+        from vertexai.preview.generative_models import ResponseBlockedError
```

Comment:
imports can be on top of the file, for this package Google dependencies are installed